mig/images/js/jquery.filemanager.js

```diff
@@ -1028,21 +1028,14 @@ if (jQuery) (function($){
             $.ajax({
                 url: download_url,
                 dataType: 'binary',
-                method: 'GET',
+                method: 'HEAD',
                 xhrFields: {
                     responseType: 'blob' // Set the response type to 'blob'
                 },
                 success: function(blob) {
                     console.info("Starting download of "+file_name)
-                    let fileURL = URL.createObjectURL(blob);
-                    var link = document.createElement('a');
-                    link.href = fileURL;
-                    /* downloaded filename */
-                    link.download = file_name;
-                    document.body.appendChild(link);
-                    link.click();
-                    URL.revokeObjectURL(fileURL);
                     stopProgress("");
```

Comment:
It seems that stopProgress() defines its own default value if none is given, so I think it would be best to delegate that to the function itself as we do in other places instead of settings an "" (empty) value